### PR TITLE
Adding CONTRIBUTING.md file.

### LINKED ISSUE #12: Adding CONTRIBUTION.md file
Adding a new CONTRIBUTION.md file to decide how users can contribute our project.

### PR DESCRIPTION
**Pull request by MatanP12**
Adding CONTRIBUTING.md file, which describes the process of contributing the ResuMe project.

fix #12

### CODE REVIEW

**Review by OmerCS8 — APPROVED** [review]
LGTM

**Comment by yulisuliman** [comment]
Looks Great!

**Review by halleelyasaf — COMMENTED** [review]
I would add who is approved to do review and how can do merges

**Comment by MatanP12** [comment]
> I would add who is approved to do review and how can do merges

How should I write it? 
something like: "any team member and approve the PR, and only the mentors can merge it"?

**Review by yulisuliman — APPROVED** [review]
Looks great!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# CONTRIBUTION
+## Codding standard
+We expect all Python code to conform to the [PEP8](https://www.python.org/dev/peps/pep-0008) standard.
+## Pull Requests(PR)
+ - Each Pull Request should focus on single repository principle.
+ - **The Pull Request should not break any of the existing functionality**
+ - <ins>Description-</ins> the PR description should explain what changes have been made, and why. In addition, they should include a link to the relevant `issue`
+ - Each PR should include 1-5 `commits`
+ - Make sure the PR passes CI checks.
+ - Each pull request requires approves of at least <ins>2 team members</ins> before merge.
```
Comment (natifridman) [inline CONTRIBUTING.md:10]:
```suggestion
 - Each pull request requires the approval of at least <ins>2 team members</ins> before merging.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+## Getting started
+### Prerequisites
+ - Install [Vagrant](https://www.vagrantup.com/downloads).
+ - Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads).
+### Initializing virtual environment 
+1. Clone the [ResuMe](https://github.com/redhat-beyond/ResuMe) repository Using the command `git clone https://github.com/redhat-beyond/ResuMe.git`.
```
Comment (natifridman) [inline CONTRIBUTING.md:16]: You need to create a fork and a separate branch for development.

**Review by natifridman — COMMENTED** [review]
There are some things that I think that missing here:
- How should issues look like? any things that we would like to see in the issues?
- You need to update the ReadMe if needed
- Usually, there is a code of conduct in the `CONTRIBUTING.md` file. You can see this [example contributing file](https://gist.github.com/PurpleBooth/b24679402957c63ec426) and [Github contributing guide](https://github.com/github/docs/blob/main/CONTRIBUTING.md) for reference

File: CONTRIBUTING.md (hunk 1 above)
Comment (halleelyasaf) [inline CONTRIBUTING.md:10]: How about this?
 Each pull request requires approves of at least <ins>2 team members</ins> before merge.
 The team members of ResuMe are:
- Omer Shor (github user
- Yuli (github user)
- Matan  (github user)
- etc

**Review by AlonShakaroff — APPROVED** [review]
Looks good to me

**Review by halleelyasaf — APPROVED** [review]
LGTM

**Review by TomerNewmanPrograms — APPROVED** [review]
Looks Great!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+
+# CONTRIBUTING
+When contributing to this repository, please first discuss the change you wish to make via issue, GitHub, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```
Comment (abiton1) [inline CONTRIBUTING.md:5]:
```suggestion
Please follow our code of conduct in all your interactions with the project.
```

**Review by abiton1 — APPROVED** [review]

**Review by natifridman — APPROVED** [review]
Approved.
Please use meaningful commit messages, and not just "changing line x".

Please squash your commits so we can merge it

**Comment by MatanP12** [comment]
> Approved. Please use meaningful commit messages, and not just "changing line x".
> 
> Please squash your commits so we can merge it

Squashed :)

**Comment by abiton1** [comment]
It fails with this error:
`An error occurred while installing pycodestyle==2.8.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4' --`

To rerun it, you can click the `Actions` tab at the top.
Click the failed test and then click `Re run all jobs` at the top right

**Pull request merged**